vulnerable_compilable.cpp

```
@@ -0,0 +1,110 @@
+    VulnerableServer() {
+        adminUser = "admin";
+        adminPass = "password123";
+    }
+
+    // 模擬一個綜合性服務流程
+    void run() {
+        // 從檔案讀取指令，但未檢查檔案是否存在、也沒有驗證輸入長度
+        char inputCommand[10] = {0};
+        ifstream infile("/nonexistent_file.txt");
+        // 警告：本來指定複製長度超出陣列大小，這是漏洞的故意設計
+        infile.getline(inputCommand, 50);
```

Cppcheck (2.10-2)

[error] Buffer is accessed out of bounds

(bufferAccessOutOfBounds)